### ISSUE TITLE
1.16: Discord integration toggle gets overridden by channel list settings.

### ISSUE THREAD

**Issue by mibby** [kangarko/ChatControl#420]
Minecraft Version: 1.16.4
MySQL: true
BungeeCord: false

ChatControl Red 10.1.12

When doing some testing with discord integration disabling, I noticed that the integration section gets overridden by the channels list setup section if you have ``Discord: true``. This causes messages to send twice to discord by both DiscordSRV & ChatControl.

Shouldn't the ``Integration`` section setting take priority and globally disable hooking into DiscordSRV?

```
Channels:

  List:
    standard:
      Format: chat
      Discord: true
    staff-chat:
      Format: admin-chat
      Discord: true
```

```
Integration:
  # Integration with DiscordSRV. 
  Discord:
    
    # Enable all integration with DiscordSRV?
    Enabled: false
    
    # Discord does not let us edit messages, so we must remove the original message and resend
    # it as the bot. You will see your message disappear and reappear in a second or less. 
    # true = messages are sent as bot (username icons not visible)
    # false = we do not edit your discord messages (warning: this breaks antispam, rules and 
    #         removing messages on discord)
    Send_Messages_As_Bot: false
  
    # What chat channels should we map when sending/receiving a message to/from Discord?
    # Write <channel id>: <channel name> (the channel name must equal your ChatControl channel
    #                                     name, but not necessarily the one on Discord)
    Connected_Channels:
      280012581983354881: standard
      494635905013907457: staff-chat
```

Had to disable discord in both sections for ChatControl to stop hooking.

**Comment by kangarko** [comment]
@ElBananaa can you please have a look? Also please wait and test the next upcoming version :)

**Comment by ElBananaa** [comment]
> @ElBananaa can you please have a look? Also please wait and test the next upcoming version :)

Hey!

So, I tried to reproduce this issue on a completely fresh install with only ChatControl-Red, its dependencies, Luckperms and DiscordSRV, they're all up-to-date. I could actually reproduce this issue. Messages from the game are sent twice to discord: One sent by DiscordSRV, and one sent by ChatControl.

When testing this, I was in the admin-chat channel in read-mode, and in the standard channel in write mode. I then tried to completely leave the standard channel, and join the admin-chat channel in write mode, and in this case, my message is only sent once to a different channel on discord.

I tried the same, but with the standard channel, and the message is, this time, sent twice, even if I'm only in one channel.
I also tried to send a message in this channel using `/chc send standard test` and this time, my message was only sent once.
